context_test.go

```diff
@@ -1,15 +1,17 @@
 package xecho
 
 import (
-	"github.com/labstack/echo"
-	"github.com/newrelic/go-agent"
-	"github.com/newrelic/go-agent/_integrations/nrlogrus"
-	"github.com/sirupsen/logrus"
-	"github.com/stretchr/testify/assert"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
+
+	"github.com/labstack/echo"
```

Comment:
Really minor thing, but we made a decision to just use go fmt in this lib, and I can see you have used go imports. If we use both we are going to get a horrible mix of the way imports are split up - perhaps we can revert to just having a single block of imports?

Reply:
Ok, will revert.